less/mixins.less

```diff
@@ -1 +1,16 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+
+.custom-font(@size, @weight) {
+    font-size: @size;
+    font-weight: @weight;
+}
```

Comment:
GREAT use case for a parametric mixin, and you named it something future devs can easily understand. Awesome.